src/EventHandler/ConfigurableEventHandlerEntityBundle.php

```diff
@@ -16,6 +16,40 @@
 class ConfigurableEventHandlerEntityBundle extends ConfigurableEventHandlerBase {
 
   /**
+   * The bundles information for the entity.
+   *
+   * @var array
+   */
+  protected $bundlesInfo;
+
+  /**
+   * The entity info plugin definition.
+   *
+   * @var mixed
+   */
+  protected $entityInfo;
+
+  /**
+   * The entity type.
+   *
+   * @var string
+   */
+  protected $entityType;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(array $configuration, $plugin_id, $plugin_definition) {
+    parent::__construct($configuration, $plugin_id, $plugin_definition);
+    $this->entityType = $this->getEventNameSuffix();
```

Comment:
damn... it should be base name. so we do not need getEventNameSuffix, but getEventBaseName instead, right?

we get bundle from form state.

I do not like that getEventBaseName is in the rules event manager. I can not call it easily from here... What would you suggest?